test/expected.txt

```diff
@@ -198,6 +201,21 @@
 || Enter command: || [Command entered:  viewall 5]
 || Viewing person: Esther Potato Phone: 555555 Email: esther@not.a.real.potato Address: (private) 555, epsilon street Tags: [tubers][starchy]
 || ===================================================
+|| Enter command: || [Command entered:  sort a]
+|| Address book has been sorted by ascending order!
+|| ===================================================
```

Comment:
Interesting that you chose to catch for trailing arguments (good design to do so), however, your feedback message still indicates a success, which muddles the true outcome